src/content/blog/2024_Q3.md

```diff
@@ -0,0 +1,99 @@
+author: Kallie Ferguson
+day: 15
+month: 10
+year: 2024
+tags: 
+  - Quarterly Updates
+---
+
+Dear Unitary Fund community,
+
+We had a busy Q3 in 2024 and we are excited to share our highlights! This quarter we launched the third annual 2024 QOSS Survey. The annual survey is a state of the community & industry snapshot. It is a chance for anyone in the quantum computing industry to share their voice in the development of our field. The survey covers information on demographics, experience, community, research, tech stacks and more.
+
+If you are a user or developer of software for any kind of quantum technology, we kindly encourage you to take this survey and contribute your voice to the future of quantum computing. Please note the survey will be available through the end of October 31, 2024. [**Take the survey here**](https://www.surveymonkey.com/r/qosssurvey24).
+
+This September we also wrapped up our second annual [unitaryCON](https://unitary.fund/community/unitaryCON/), a collaborative workshop for the extended Unitary Fund community. With the help of Dr. Alexandru Paler, unitaryCON 2024 was hosted at Aalto University in Espoo, Finland. UnitaryCON provided a unique opportunity for Unitary Fund advisors, grantees, staff, ambassadors and friends to gather, talk, learn, and share ideas about how to move the field forward. With an event rating of 9.36 out of 10, the event was a success in bringing the community together to learn and network. Read more about the event and find links to talk information and slides [here](https://unitary.fund/posts/2024_unitarycon/). 
+
+We look forward to another exciting quarter to close out 2024, and thank you to all of our community members for the impactful contributions to the open source quantum ecosystem in Q3!
+Make sure to follow our [Discord](https://discord.com/invite/JqVGmpkP96), [X](https://twitter.com/unitaryfund), [LinkedIn](https://www.linkedin.com/company/unitary-fund/), and our [Community Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_mgqdq6hj2isi4d6h467kfqvg60@group.calendar.google.com).
+
+## New from Unitary Fund 
+
+**Unitary Fund Research**: [W. J. Zeng](https://arxiv.org/search/quant-ph?searchtype=author&query=Zeng,+W+J), [F. Labib](https://arxiv.org/search/quant-ph?searchtype=author&query=Labib,+F), [V. Russo](https://arxiv.org/search/quant-ph?searchtype=author&query=Russo%2C+V), *Towards violations of Local Friendliness with quantum computers*. arXiv preprint (2024), [[2409.15302](https://arxiv.org/abs/2409.15302)]
+
+**Mitiq**
+
+[v0.39.0](https://github.com/unitaryfund/mitiq/releases/tag/v0.39.0): updates to our documentation, beginning with the completion of the first section of the Pauli Twirling user guide, which offers a comprehensive introduction to this feature. Additionally, we've added a new tutorial on CDR (Clifford Data Regression) using [Qrack](https://github.com/unitaryfund/qrack/) as an efficient near-Clifford simulator.
```

Comment:
@natestemen this is not all we want to say about Mitiq, is it?

Reply:
I had already run by this Nate last week, but happy to add more if there are things to be included.

Reply:
Since we have some time should we add a note about the Mitiq workshop at UMass Amherst, and the Mitiq tutorial at IEEE QCE? I was thinking that the update was purely about code changes, but maybe it makes sense to highlight other things as well?

We do have another release of Mitiq cut, but since it was released last week, we should include it in the Q4 post.